pom.xml

```diff
@@ -193,6 +193,32 @@
           </systemProperties>
         </configuration>
       </plugin>
+
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-failsafe-plugin</artifactId>
+          <version>3.0.0-M2</version>
+          <configuration>
+              <useSystemClassLoader>false</useSystemClassLoader>
+              <systemProperties>
+                  <property>
+                      <name>sqlite4java.library.path</name>
+                      <value>${project.build.directory}/test-lib</value>
+                  </property>
+              </systemProperties>
+              <includes>
+                  <include>**/*ITCase.java</include>
```

Comment:
What is this matching/including?

Reply:
This includes any java file that ends in ITCase.java to run as an integration test

Reply:
(So it excludes *Test.java, for example)

Reply:
OK, so it sounds like this is intentionally matching nothing right now, which is one of the things I was wondering about. Otherwise LGTM then.